packages/phone-number-privacy/common/src/interfaces/requests.ts

```diff
@@ -23,35 +23,12 @@ import {
 // of interface. Otherwise the compiler complains about a missing index signature.
 // tslint:disable:interface-over-type-literal
 
-export enum PhoneNumberPrivacyEndpoint {
-  STATUS = '/status',
-  METRICS = '/metrics',
-  GET_BLINDED_MESSAGE_PARTIAL_SIG = '/getBlindedMessagePartialSig',
-  GET_QUOTA = '/getQuota',
-}
-
-export enum DomainEndpoint {
-  DISABLE_DOMAIN = '/domain/disable',
-  DOMAIN_SIGN = '/domain/sign/',
-  DOMAIN_QUOTA_STATUS = '/domain/quotaStatus',
-}
-
-export type SignerEndpoint = PhoneNumberPrivacyEndpoint | DomainEndpoint
-export const SignerEndpoint = { ...PhoneNumberPrivacyEndpoint, ...DomainEndpoint }
-
-export enum CombinerEndpoint {
-  SIGN_MESSAGE = '/getBlindedMessageSig',
-  MATCHMAKING = '/getContactMatches',
-}
-
-export type Endpoint = SignerEndpoint | CombinerEndpoint
-export const Endpoint = { ...SignerEndpoint, ...CombinerEndpoint }
-
 export enum AuthenticationMethod {
   WALLET_KEY = 'wallet_key',
   ENCRYPTION_KEY = 'encryption_key',
 }
 
+// TODO(Alec): investigate alias in sdk/identity
```

Comment:
Context: I created those aliases when I was deduplicating request types across phone-number-privacy-common and the identity SDK. They were duplicate definitions, and I maintained them as an alias for backwards compatibility in case any code was relying on those types from the SDK.